vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -45,7 +45,30 @@ module private SessionHandling =
                   { new IQuickInfoSource with
                     member __.AugmentQuickInfoSession(session,_,_) = currentSession <- Some session
                     member __.Dispose() = () }
-    
+
+type internal SourceLink(run) as this = 
+    inherit Documents.Hyperlink(run)
+    let lessOpacity =
+      { new IValueConverter with
+        member this.Convert(value, targetType, _, _) =
+            match value with 
+            | :? Color as c when targetType = typeof<Color> ->
+                // return same color but slightly transparent
+                Color.FromArgb(90uy, c.R, c.G, c.B) :> _
+            | _ -> DependencyProperty.UnsetValue
+        member this.ConvertBack(_,_,_,_) = DependencyProperty.UnsetValue }
+    let underlineBrush = Media.SolidColorBrush()
+    do BindingOperations.SetBinding(underlineBrush, SolidColorBrush.ColorProperty,
+                                    Binding("Foreground.Color", Source = this, Converter = lessOpacity)) |> ignore
+    let slightUnderline = TextDecoration(Location = TextDecorationLocation.Underline, PenOffset = 1.0, Pen = Pen(Brush = underlineBrush))
+    do this.TextDecorations <- TextDecorationCollection [slightUnderline]
+
+    override this.OnMouseEnter(e) = 
+        base.OnMouseEnter(e)
+        this.TextDecorations <- TextDecorations.Underline
```

Comment:
I'm now wondering whether it would be better if we always underlined the hyperlinks, but say at 30/40% opacity. Then when the mouse is hovered over the tooltip, it becomes 90%?

I say this as otherwise the discoverability of this feature is very poor.